File: Backend/Iteramico/DomainRepository/Models/Country.cs
namespace DomainRepository.Models
{
#nullable disable
    public class Country
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string LocalName { get; set; }
        public string CountryCode { get; set; }

        public virtual List<City> Cities { get; set; }
    }
}

File: frontend/lib/models/country.dart
class Country {
  final int id;
  final String name;
  final String localName;
  final String countryCode;

  Country(
      {required this.id,
      required this.name,
      required this.localName,
      required this.countryCode});

  factory Country.fromJson(Map<String, dynamic> json) {
    return Country(
      id: json['id'],
      name: json['name'],
      localName: json['localName'],
      countryCode: json['countryCode'],
    );
  }
}
